src/client/index.js

```diff
@@ -888,16 +888,16 @@ class KavaClient {
 
   /**
    * Claim Hard protocol reward using a specific multiplier and denoms
-   * @param {String} multiplierName the multiplier to claim with, such as 'small' or 'large'
    * @params {Array} choose which denom(s) of your rewards that you would like to claim
+   * and at which multiplier you claim them
+   * [{ denom: 'hard', multiplierName: 'large'}, { denom: 'ukava', multiplierName: 'small' }]
```

Comment:
just going to comment this once though it's literally the same change to different messages in this file. these rewards take a wallet address and a `denomToClaim` which is represented in the example above. usdxMinting claims and usdxMinting validator vesting claims do not follow this pattern, but hard, swap, and delegator related claims do